src/portalbackend/PortalBackend.DBAccess/Models/CompanyServiceAccountDetailedData.cs

```diff
@@ -33,8 +33,11 @@ public record CompanyServiceAccountDetailedData(
     CompanyServiceAccountTypeId CompanyServiceAccountTypeId,
     Guid? SubscriptionId,
     ConnectorResponseData? ConnectorData,
-    OfferResponseData? OfferSubscriptionData);
+    OfferResponseData? OfferSubscriptionData,
+    CompanyLastEditorData? CompanyLastEditorData);
 
 public record ConnectorResponseData(Guid Id, string Name);
 
 public record OfferResponseData(Guid Id, OfferTypeId Type, string? Name, Guid? SubscriptionId);
+
+public record CompanyLastEditorData(string Name, string CompanyName);
```

Comment:
the lastname of a company user is nullable therefore we should change the name to be nullable here as well.

```suggestion
public record CompanyLastEditorData(string? Name, string CompanyName);
```